tools/vipps-qr-api-postman-collection.json

```diff
@@ -93,7 +89,6 @@
 						},
 						"url": {
 							"raw": "{{base_url}}/qr/v1/merchant-redirect",
-							"protocol": "https",
```

Comment:
Guess this should not be deleted

Reply:
This was actually breaking it because the variable {{base_url}} included "https" and it showed up in postman as https://https://